src/api/clinical/medication/medication.consumption/medication.consumption.auth.ts

```diff
@@ -79,8 +79,8 @@ export class MedicationConsumptionAuth {
     static getSummaryForDay: AuthOptions = {
         ...DefaultAuthOptions,
         Context     : `${this._baseContext}.GetSummaryForDay`,
-        Ownership   : ResourceOwnership.Owner,
-        ActionScope : ActionScope.Tenant,
+        Ownership   : ResourceOwnership.System,
```

Comment:
Medication consumption is for a particular patient. 
Why this was changed?

Reply:
Changed.This endpoint is allowed for all kinds of user. That is the resaon I excluded the consent of resource owner.